src/material-examples/datepicker-filter/datepicker-filter-example.ts

```diff
@@ -0,0 +1,10 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'datepicker-filter-example',
+  templateUrl: 'datepicker-filter-example.html',
+  styleUrls: ['datepicker-filter-example.css'],
+})
+export class DatepickerFilterExample {
+  myFilter = (d: Date) => d.getFullYear() > 2017;
```

Comment:
Miles means that the example for filtering should show something else (e.g., filter out weekends).